### ISSUE TITLE
Step Indicator dynamic content not accessible by keyboard

### ISSUE THREAD

**Issue by louiseClarkUSDS** [usds/techfar-hub-website-v3#57]
### Accessibility Issue

In order to meet WCAG 2.0 Level A Success Criteria "[Keyboard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)" , the step indicator dynamic content located on the "[Get Started](https://techfarhub.usds.gov/get-started/)" page must be available to people who use a keyboard only. Currently, the only way for a person to access the hidden content in the step indicator is via a mouse click.  A person using a keyboard only can't access the content for the step indicator beyond the first step. The step indicator isn't in the tab index because there aren't any interactive elements. 

### Why this matters

User interfaces must be entirely operable by people using a keyboard only. Ensuring keyboard operability means that the UI includes, for example, people with low vision or those with tremors who may have difficulty using a mouse to interact with specific elements.  

### How to fix

This is up to the designer and developer as to how to handle implementation. It may be to use a different component vs a step indicator? It may be that the current step indicator can be refactored to allow for keyboard accessibility of dynamic content? The expectation is that a keyboard only user can access all hidden/dynamic content.

**Comment by mattbowen-usds** [comment]
This was resolved in PR #58
